app/assets/stylesheets/globals/variables/_bootstrap.scss

```diff
@@ -0,0 +1,866 @@
+// $badge-line-height:           1 !default;
+// $badge-border-radius:         10px !default;
+
+
+// //== Breadcrumbs
+// //
+// //##
+
+// $breadcrumb-padding-vertical:   8px !default;
+// $breadcrumb-padding-horizontal: 15px !default;
+// //** Breadcrumb background color
+// $breadcrumb-bg:                 #f5f5f5 !default;
+// //** Breadcrumb text color
+// $breadcrumb-color:              #ccc !default;
+// //** Text color of current page in the breadcrumb
+// $breadcrumb-active-color:       $gray-light !default;
+// //** Textual separator for between breadcrumb elements
+// $breadcrumb-separator:          "/" !default;
+
+
+// //== Carousel
+// //
+// //##
+
+// $carousel-text-shadow:                        0 1px 2px rgba(0,0,0,.6) !default;
+
+// $carousel-control-color:                      #fff !default;
+// $carousel-control-width:                      15% !default;
+// $carousel-control-opacity:                    .5 !default;
+// $carousel-control-font-size:                  20px !default;
+
+// $carousel-indicator-active-bg:                #fff !default;
+// $carousel-indicator-border-color:             #fff !default;
+
+// $carousel-caption-color:                      #fff !default;
+
+
+// //== Close
+// //
+// //##
+
+// $close-font-weight:           bold !default;
+// $close-color:                 #000 !default;
+// $close-text-shadow:           0 1px 0 #fff !default;
+
+
+// //== Code
+// //
+// //##
+
+// $code-color:                  #c7254e !default;
+// $code-bg:                     #f9f2f4 !default;
+
+// $kbd-color:                   #fff !default;
+// $kbd-bg:                      #333 !default;
+
+// $pre-bg:                      #f5f5f5 !default;
+// $pre-color:                   $gray-dark !default;
+// $pre-border-color:            #ccc !default;
+// $pre-scrollable-max-height:   340px !default;
+
+
+// //== Type
+// //
+// //##
+
+// //** Horizontal offset for forms and lists.
+// $component-offset-horizontal: 180px !default;
+// //** Text muted color
+// $text-muted:                  $gray-light !default;
+// //** Abbreviations and acronyms border color
+// $abbr-border-color:           $gray-light !default;
+// //** Headings small color
+// $headings-small-color:        $gray-light !default;
+// //** Blockquote small color
+// $blockquote-small-color:      $gray-light !default;
+// //** Blockquote font size
+// $blockquote-font-size:        ($font-size-base * 1.25) !default;
+// //** Blockquote border color
+// $blockquote-border-color:     $gray-lighter !default;
+// //** Page header border color
+// $page-header-border-color:    $gray-lighter !default;
+// //** Width of horizontal description list titles
+// $dl-horizontal-offset:        $component-offset-horizontal !default;
+// //** Point at which .dl-horizontal becomes horizontal
+// $dl-horizontal-breakpoint:    $grid-float-breakpoint !default;
+// //** Horizontal line color.
+// $hr-border:                   $gray-lighter !default;
```

Comment:
:-1:

Dead code should not be checked in. Si tu mets à jour une dépendance qui renomme ces variables commentées, elles ne fonctionneront plus et tu ne le sauras qu'en les décommandant.